src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
+    if (value || value === 0) {
```

Comment:
```suggestion
    if (value !== undefined) {
```
You can just check if the value is not undefined